Modules/book.js

```diff
@@ -0,0 +1,7 @@
+// eslint-disable-next-line max-classes-per-file
```

Comment:
Please don't disable linters until you really need to disable them. Here, there is no linter error when you delete this line. Please delete this line. 👍 
There are a few reasons for using linters:
Catching syntax errors is more efficient. There is no need to debug simple mistakes like typos - the linter does it for you.
The entire codebase looks like written by one person.
Programmers can focus on solving problems, instead of cleaning up the code.

You can search for the linter error on their [documentation page](https://eslint.org/docs/latest/) and solve it.